vizro-core/src/vizro/models/_components/form/dropdown.py

```diff
@@ -141,7 +151,7 @@ def __call__(self, options):
 
         return html.Div(
             children=[
-                dbc.Label(self.title, html_for=self.id) if self.title else None,
+                dbc.Label([self.title, *description], html_for=self.id) if self.title else None,
```

Comment:
If we make the following change, would it be easier for users to move the icon with the tooltip to float right? 
```suggestion
                dbc.Label([self.title], html_for=self.id) if self.title else None,
                dbc.Label([*description], html_for=self.id) if self.title else None,
```

Reply:
I just pushed some changes where I turned the page-title, dashboard-title, and form-label into flexboxes. Not only does this fix the centering issue @antonymilne pointed out, but it also makes it super easy to position the icon on the right using some custom CSS, without needing any further changes here 👍 

You could just do:

```
.form-label {
    justify-content: space-between;
}
```

And then you get this:

![Screenshot 2025-04-04 at 14 53 38](https://github.com/user-attachments/assets/518b4ad1-01b4-454d-af97-2dd46c2a85cc)

Reply:
Nice, this is a perfect solution I think 👍